app/models/stored_value.rb

```diff
@@ -0,0 +1,23 @@
+require 'base64'
+
+##
+# This class represents temporary storage for values processed in background.
+# This is a database solution, wich should not be the only one, as it has many limitations.
+
```

Comment:
I don't like empty lines between docs and the code, I think at least for consistency reasons, we shouldn't do it :-)